tests/runtest.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+if [ ! "$(dpkg -l | grep socat)" ]; then
```

Comment:
I would not bother with this, either it is installed and everything is OK, or it is not and script will fail anyway. You are not checking whether `docker` is installed, why would you special case socat?